lib/google/ads/google_ads/config.rb

```diff
@@ -31,6 +31,7 @@ class Config
 
         attr_accessor :developer_token
         attr_accessor :login_customer_id
+        attr_accessor :linked_customer_id
```

Comment:
While you're implementing this, could you also actually update the code to support it? It should be treated pretty much exactly the same as login_customer_id, passed along to the API as a header.

Reply:
Yup sure. Added the support.